crates/polars-ops/src/series/ops/various.rs

```diff
@@ -10,15 +10,15 @@ use crate::series::ops::SeriesSealed;
 pub trait SeriesMethods: SeriesSealed {
     /// Create a [`DataFrame`] with the unique `values` of this [`Series`] and a column `"counts"`
     /// with dtype [`IdxType`]
-    fn value_counts(&self, multithreaded: bool, sorted: bool) -> PolarsResult<DataFrame> {
+    fn value_counts(&self, sort: bool, parallel: bool) -> PolarsResult<DataFrame> {
         let s = self.as_series();
         // we need to sort here as well in case of `maintain_order` because duplicates behavior is undefined
-        let groups = s.group_tuples(multithreaded, sorted)?;
+        let groups = s.group_tuples(parallel, sort)?;
         let values = unsafe { s.agg_first(&groups) };
         let counts = groups.group_lengths("counts");
         let cols = vec![values, counts.into_series()];
-        let df = DataFrame::new_no_checks(cols);
```

Comment:
This was the issue - a check is needed on the column name as it CAN contain a duplicate.

Reply:
How about directly check self.name() and ＂counts" to avoid other overhead introduced by DataFrame::new. But it shouldn't cost mush as we only have two columns here，it's up tp you :）

Reply:
That would be more efficient! Can do that.